apps/aecore/src/aec_dev_reward.erl

```diff
@@ -20,34 +21,47 @@
 -define(TOTAL_SHARES, 1000). % 100 shares == 10% of the reward
 %%% weighted avg BRI voting result of 1% to 20% (yes) votes is 10.89869526640124746202%, 109 of 1000 shares will be the protocol reward
 %%% for: "ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8:109"
--define(MAINNET_BENEFICIARIES, [{<<172,241,128,85,116,104,119,143,197,105,4,192,224,207,200,138,230,84,111,38,89,33,239,21,201,183,185,209,19,60,109,136>>, 109}]).
+%%%%%%%%% TODO update with real values
```

Comment:
Is this a TO DO for later or something that was planned for this PR?

Reply:
This is for later, it should be before updated before Ceres goes live but we are waiting for a new BRI account from the foundation.